sdk/python/pulumi_aws/iam/_enums.py

```diff
@@ -151,6 +151,9 @@ class ManagedPolicy(str, Enum):
     AMAZON_EC2_SPOT_FLEET_AUTOSCALE_ROLE = "arn:aws:iam::aws:policy/service-role/AmazonEC2SpotFleetAutoscaleRole"
     AMAZON_EC2_SPOT_FLEET_ROLE = "arn:aws:iam::aws:policy/service-role/AmazonEC2SpotFleetRole"
     AMAZON_EC2_SPOT_FLEET_TAGGING_ROLE = "arn:aws:iam::aws:policy/service-role/AmazonEC2SpotFleetTaggingRole"
+    AMAZON_EC_S_FULL_ACCESS = "arn:aws:iam::aws:policy/AmazonECS_FullAccess"
```

Comment:
> AMAZON_EC_S_FULL_ACCESS

@komalali another concern is this name for python enum. Is it expected?

Reply:
Yeah, this is an unfortunate side-effect of the camel case to snake case algorithm.  I updated this enum name to `AmazonECSFullAccess` which creates the desired python enum.